src/features/groups/components/InviteMemberDialog/InviteMemberDialogContent.tsx

```diff
@@ -0,0 +1,46 @@
+import { LinkIcon } from '@/assets/images/icons';
+import {
+  Button,
+  DialogBody,
+  DialogContent,
+  DialogHeader,
+  DialogTitle,
+  Icon,
+} from '@/components/ui';
+
+import { useEffect, useState } from 'react';
+
+export const InviteMemberDialogContent = ({ isDialogOpen }: { isDialogOpen: boolean }) => {
+  const [isCopyDisabled, setIsCopyDisabled] = useState(false);
+  const handleLinkCopy = () => {
+    setIsCopyDisabled(true);
+  };
+  const handleTextChange = (isClicked: boolean) => {
+    return isClicked ? 'Copied!' : 'Copy Link';
+  };
+  useEffect(() => {
+    setTimeout(() => setIsCopyDisabled(false), 200);
```

Comment:
@nick-y-ito 
Thanks for re-reviewing, and sorry for not explaining why I did thi way.

in this approach, my intention was to remain "copied" text until the dialog disappears.
Without `setTimeout`, the text would change to "Copy Link" before dialog disappears as video capture below.

[linkDialog_wo_setTimeout.webm](https://github.com/nishiki-tech/nishiki-frontend/assets/142355969/d06d91f8-879d-4e06-a03d-77aeb77fdfd1)

So, I thought I needed to add `setTimeout`  in order to solve this awkward UI.

Reply:
One more thing , maybe I should change the name `isCopyDisabled` and `setIsCopyDisabled` to `isLinkButtonClicked` `setIsLinkButtonClicked` in order to improve readability.

Reply:
@ShoeheyOt 
Thanks for clarifying! 

> Also, it's a minor point, but if your intention is to disable it after the dialog is completely closed, it would be better to match the duration of the animation defined in tailwind.config.ts. Nonetheless, it already looks visually perfect.

What I meant here was (sorry for the confusion): 

> ... if your intention is to delay enabling the button until the dialog is completely closed, ...

So, our understanding was the same!

Therefore, please consider it matching the duration defined in `tailwind.config.ts` and adding a comment to describe your intention. When using the `setTimeout` I belive it is important to have a reasonable reason why you have chosen the duration.

Reply:
@nick-y-ito 
OK! Thank you.
I changed duration time from 0.2 to 0.3 as same as fadeIn/fadeOut, and also left comment. If I'm doing wrong, please let me know.